xml/admin_nfsganesha.xml

```diff
@@ -9,26 +9,13 @@
  xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
  xml:id="cha-ceph-nfsganesha">
 <!-- ============================================================== -->
- <title>&ganesha;: Export &ceph; Data via NFS</title>
```

Comment:
We need this &lt;info> block there

Reply:
Why? It's not in other files. It seems very random.

Reply:
i believe that for example files without the translation flag wont be included in the localization set

Reply:
Okay, no problem. We might need to go through the individual files and make sure they're all indicated. @fsundermeyer is this definitely the case?